### PR TITLE
US114948 - Remove overall-score-flag

### PR DESCRIPTION
**Pull request by max-hao-d2l**
[US114948](https://rally1.rallydev.com/#/detail/userstory/378529980860?fdp=true): LD Flag Cleanup - rubrics-overall-score

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -120,14 +120,6 @@ <h3>Assessed Rubric with multiple criteria groups</h3>
 	<demo-snippet>
 		<d2l-rubric read-only="" href="data/rubrics/organizations/multiple-groups/203.json" token="foozleberries" assessment-href="data/rubrics/organizations/multiple-groups/203/assessment/10.json"></d2l-rubric>
 	</demo-snippet>
-	<h3>Rubric with Overall Score</h3>
```
Comment (mpharoah-d2l) [inline demo/index.html]: We still want this test, right? But without the flag passed in?

Comment (max-hao-d2l) [reply demo/index.html]: Fixed

File: test/components/d2l-rubric-text-only.js [hunk 2]
```diff
@@ -24,13 +24,5 @@ suite('<d2l-rubric-text-only>', function() {
 			outOfContainer = dom(element.root).querySelector('.out-of-container');
 			expect(outOfContainer.attributes).to.have.ownProperty('hidden');
 		});
-
-		test('Overall Score section is not rendered if the rubric has no overall score', function(done) {
```
Comment (mpharoah-d2l) [inline test/components/d2l-rubric-text-only.js]: Why was this test removed?

Comment (rgoessl) [reply test/components/d2l-rubric-text-only.js]: The fixture wasn't passed the flag, so I can see it being useless now

Comment (rgoessl) [reply test/components/d2l-rubric-text-only.js]: Though you're right, there are many more conditions for displaying overall all, this should be re-added

Comment (max-hao-d2l) [reply test/components/d2l-rubric-text-only.js]: Holistic rubrics should be the only case where the overall level does not show up. I've added a test to check if the overall level does not show up for holistic rubrics.

**Pull request merged**